test/functional/read_directory.js

```diff
@@ -1,7 +1,7 @@
 const Category = require('../../lib/category');
 const MarkdownRenderer = require('../../lib/md-renderer');
 const path = require('path');
-const createDummyConfig = require('../unit/zendesk-uploader/test-utils').createDummyConfig;
+const createDummyConfig = require('../unit/test-utils').createDummyConfig;
```

Comment:
I don't think so...I'm not getting any errors from tests. I moved all test-utils to `unit/test-utils`.